samples/outlook/30-recipients-and-attendees/get-bcc-message-compose.yaml

```diff
@@ -0,0 +1,64 @@
+order: 7
+id: outlook-recipients-and-attendees-get-bcc-message-compose
+name: Get bcc (Message Compose)
```

Comment:
are you planning on adding samples to add the recipients as well?

Reply:
yes, I'll need to do add and remove for to, cc, and bcc